RecoEgamma/EgammaElectronProducers/plugins/GsfElectronBaseProducer.cc

```diff
@@ -39,21 +85,65 @@ namespace {
 
 }  // namespace
 
-using namespace reco;
+class GsfElectronBaseProducer : public edm::stream::EDProducer<edm::GlobalCache<GsfElectronAlgo::HeavyObjectCache>> {
```

Comment:
Because we would have to rename the file which might be confusing when comparing to older CMSSW versions. But if you agree that the plugin should be renamed to `GsfElectronProducer`, I will do this or take another name that you suggest.

Reply:
OK, let's just drop the "Base" both in the class and the file.
Thank you.